app/assets/stylesheets/components.scss

```diff
@@ -1217,10 +1240,9 @@ a.status__content__spoiler-link {
   }
 
   .column, .drawer {
-    width: 350px;
     border-radius: 4px;
-    height: 90vh;
-    margin-top: 5vh;
+    height: 100vh;
+    margin: 0;
```

Comment:
Before only `margin-top` was applied but now we're applying margin to all sides. If you don't need to apply margin to all sides, you should avoid doing so.